specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2023-07-12-preview/AlertsManagement.json

```diff
@@ -0,0 +1,1568 @@
+      "required": true,
+      "x-ms-skip-url-encoding": true,
+      "x-ms-parameter-location": "client"
+    },
+    "subscriptionId": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The ID of the target subscription.",
+      "minLength": 1
+    },
+    "alertId": {
+      "name": "alertId",
+      "description": "Unique ID of an alert instance.",
+      "type": "string",
+      "in": "path",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "targetResourceGroup": {
+      "description": "Filter by target resource group name. Default value is select all.",
+      "name": "targetResourceGroup",
+      "type": "string",
+      "in": "query",
+      "required": false,
+      "x-ms-parameter-location": "method"
+    },
+    "targetResource": {
+      "description": "Filter by target resource( which is full ARM ID) Default value is select all.",
+      "name": "targetResource",
+      "type": "string",
+      "in": "query",
+      "required": false,
+      "x-ms-parameter-location": "method"
+    },
+    "targetResourceType": {
+      "description": "Filter by target resource type. Default value is select all.",
+      "name": "targetResourceType",
+      "type": "string",
+      "in": "query",
+      "required": false,
+      "x-ms-parameter-location": "method"
+    },
+    "monitorService": {
+      "description": "Filter by monitor service which generates the alert instance. Default value is select all.",
+      "name": "monitorService",
+      "type": "string",
+      "in": "query",
+      "required": false,
+      "x-ms-parameter-location": "method",
+      "enum": [
+        "Application Insights",
+        "ActivityLog Administrative",
+        "ActivityLog Security",
+        "ActivityLog Recommendation",
+        "ActivityLog Policy",
+        "ActivityLog Autoscale",
+        "Log Analytics",
+        "Nagios",
+        "Platform",
+        "SCOM",
+        "ServiceHealth",
+        "SmartDetector",
+        "VM Insights",
+        "Zabbix",
+        "Resource Health"
+      ],
+      "x-ms-enum": {
+        "name": "MonitorService",
+        "modelAsString": true
+      }
+    },
+    "severity": {
+      "description": "Filter by severity.  Default value is select all.",
+      "name": "severity",
+      "type": "string",
+      "in": "query",
+      "required": false,
+      "x-ms-parameter-location": "method",
+      "enum": [
+        "Sev0",
+        "Sev1",
+        "Sev2",
+        "Sev3",
+        "Sev4"
+      ],
+      "x-ms-enum": {
+        "name": "Severity",
+        "modelAsString": true
+      }
+    },
+    "smartGroupIdFilter": {
+      "description": "Filter the alerts list by the Smart Group Id. Default value is none.",
+      "name": "smartGroupId",
+      "type": "string",
+      "in": "query",
+      "required": false,
+      "x-ms-parameter-location": "method"
+    },
+    "smartGroupId": {
```

Comment:
>smartGroupId [](http://example.com/codeflow?start=5&length=12)

Can you add information to the description to tell clients what format this ID is (e.g. resourceId, guid, something else).

Reply:
It's a deprecated features which will get removed in the future.